templates/upload_error.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>CSV Upload Error</title>
+</head>
+<body>
+    <p>Error: {{ msg }}</p>
+    <a href="/mobilize/uploadcsv">Try Again</a>
```

Comment:
The href here should refer to `/docs/upload_csv.html`, as you took out this endpoint.